Index: learn/虎牙视频下载.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># 虎牙\r\n\r\nimport requests\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/learn/虎牙视频下载.py b/learn/虎牙视频下载.py
--- a/learn/虎牙视频下载.py	(revision 9ac171f81f598616cb9d560048cfca29f966022f)
+++ b/learn/虎牙视频下载.py	(date 1747122150956)
@@ -1,4 +1,9 @@
-# 虎牙
+# 虎牙视频有参数，需要注意添加
+# 头为：https://liveapi.huya.com/moment/getMomentContent
 
 import requests
 
+# 
+
+
+
